infrastructure/template.yaml

```diff
@@ -617,6 +633,20 @@ Resources:
     Properties:
       Name: !Sub ${AWS::StackName}-CheckHmrcEventBus
 
+  AuditEventResponseReceivedRule:
+    Type: AWS::Events::Rule
+    Properties:
+      EventBusName: !Ref CheckHmrcEventBus
+      Description: Capture the audit event RESPONSE_RECEIVED
+      State: DISABLED
+      EventPattern:
+        source:
+          - !Sub
+            - review-hc.${Env}.account.gov.uk
+            - Env: !Ref Environment
```

Comment:
Is this really necessary `review-hc.${Environment}.account.gov.uk` should work fine

Reply:
No this changed in https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/pull/154. I've created cascading PRs